cloud/azure/management/recovery/mode/backupjobsstatus.pm

```diff
@@ -140,6 +140,8 @@ sub check_options {
     }
 
     $self->change_macros(macros => ['warning_status', 'critical_status']);
+
+    $self->{timeframe} = (defined($self->{option_results}->{timeframe})) ? $self->{option_results}->{timeframe} : 86400;
```

Comment:
Not sure it's relevant to use the option label we used to filter metrics. As it's specific maybe name it 'lookback' but not timeframe. 

It will require to add the option locally.